src/game.py

```diff
@@ -1,46 +1,75 @@
 from board import Board
 from os import system
+from os import get_terminal_size
 
 
 class Game:
-    OBSTACLE = '# '
-    EXIT = '  '
-    PUCK = {Board.RED: 'O ', Board.BLUE: '0 ', Board.BLACK: '* '}
+    TOP_PADDING = int(get_terminal_size()[1]*0.20)
```

Comment:
At this time, seing the size and complexity of the game file, maybe spliting it would be useful
like 
a game.py for the rules mechanisms
a game_renderer.py for the display constant and methods